index_de.md

```diff
@@ -4,52 +4,52 @@ layout: signed
 
 2021-03-23
 
-Richard M. Stallman, häufig bekannt unter dem Kürzel RMS,
-war seit Jahrzehnten eine treibende Kraft in der Freien 
-Software-Bewegung, mit Beiträgen wie dem GNU Betriebssystem und Emacs.
-
-In jüngster Zeit gab verwerfliche Online-Angriffe mit dem Ziel ihn
-aus dem Vorstand der FSF zu entfernen, weil er seine persönlichen
-Meinungen zum Ausdruck gebracht hat. Wir mussten  
-dabei zuschauen wie dies bereits in organisierter Form anderen
-Aktivisten der Freien Software-Bewegung passiert ist.
-Wir werden dieses Mal nicht tatenlos dabei zusehen, wie ein Symbol 
-dieser Gemeinschaft angegriffen wird.
+Richard M. Stallman, häufig bekannt unter dem Kürzel RMS, war seit
+Jahrzehnten eine treibende Kraft in der Freien Software-Bewegung,
+mit Beiträgen wie dem GNU Betriebssystem und Emacs.
+
+In jüngster Zeit wurden eine reihe verwerflicher Online-Angriffe
+bewerkstelligt, mit dem Ziel ihn aus dem Vorstand der FSF zu entfernen,
```

Comment:
Better: "Online-Angriffe lanciert"
Meaning: "gezielt in die Öffentlichkeit gelangen lassen"